### PR TITLE
Bugfix: Correct mimetype for disabling/enabling things call

### LINKED ISSUE #1150: Disable Things via MainUI not working in latest SNAPSHOT (30.08.2021)
I have just updated my openHAB installation to the latest SNAPSHOT: openHAB3-Distribution number 2477
see https://ci.openhab.org/job/openHAB3-Distribution/2477/

Unfortunately I cannot disable a Thing via the Main UI anymore. (Doesn't matter if the thing was created as a JSON file via the UI or text based via a .things file)

It works on the console, hence I think its a bug in the UI

```
openhab> things disable tradfri:gateway:ikeagw
tradfri:gateway:ikeagw successfully disabled.
```

The error I get is: "Error while disabling or enabling: Unsupported Media Type", see Screenshot.

disableThing

I am using Chromium Version 92.0.4515.159 (Official Build) Arch Linux (64-bit)

### PR DESCRIPTION
**Pull request by t2000**
Fixes #1150

Signed-off-by: Stefan Triller <github@stefantriller.de>

### CODE REVIEW

**Comment by lolodomo** [comment]
I let @ghys review, I don't know why the media type is doubled.

**Comment by digitaldan** [comment]
> I let @ghys review, I don't know why the media type is doubled.

The first argument is the 'dataType', which is what you are expecting back from the server, the second is "contentType", the content type you are sending. 

See
https://github.com/openhab/openhab-webui/blob/4e402ef697e3dabb519b7f6ef37eb546bd236535/bundles/org.openhab.ui/web/src/js/openhab/api.js#L65

File: bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue [hunk 1]
```diff
@@ -490,7 +490,7 @@ export default {
     },
     toggleDisabled () {
       const enable = (this.thing.statusInfo.statusDetail === 'DISABLED')
-      this.$oh.api.putPlain('/rest/things/' + this.thingId + '/enable', enable.toString(), 'application/json', 'application/json').then((data) => {
+      this.$oh.api.putPlain('/rest/things/' + this.thingId + '/enable', enable.toString(), 'text/plain', 'text/plain').then((data) => {
```
Comment (digitaldan) [inline bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue:493]:
```suggestion
      this.$oh.api.putPlain('/rest/things/' + this.thingId + '/enable', enable.toString(), 'application/json', 'text/plain').then((data) => {
```

I believe the enable endpoint returns JSON, but consumes text.

 You could also just omit the last two parameters as they are the default types
```suggestion
      this.$oh.api.putPlain('/rest/things/' + this.thingId + '/enable', enable.toString()).then((data) => {
```

Comment (t2000) [reply bundles/org.openhab.ui/web/src/pages/settings/things/thing-details.vue]: Thanks, done.

**Review by ghys — CHANGES_REQUESTED** [review]
Thanks for taking care of this!
Could you please also change it here (you can enable/disable multiple things at once from the list):

https://github.com/openhab/openhab-webui/blob/cf9ae4a06b92752ac2a0deb2ee915ab23774f767/bundles/org.openhab.ui/web/src/pages/settings/things/things-list.vue#L279

**Comment by t2000** [comment]
> Could you please also change it here (you can enable/disable multiple things at once from the list):
> 
> https://github.com/openhab/openhab-webui/blob/cf9ae4a06b92752ac2a0deb2ee915ab23774f767/bundles/org.openhab.ui/web/src/pages/settings/things/things-list.vue#L279

Done

**Review by ghys — APPROVED** [review]
Thanks again!

**Pull request merged**